src/clib-search.c

```diff
@@ -108,12 +108,16 @@ static char *wiki_html_cache() {
 
   debug(&debugger, "setting cache from %s", CLIB_WIKI_URL);
   http_get_response_t *res = http_get(CLIB_WIKI_URL);
-  if (!res->ok)
+
+  if (!res->ok) {
+    http_get_free(res);
     return NULL;
+  }
 
   char *html = strdup(res->data);
   if (NULL == html)
     return NULL;
```

Comment:
We'll also leak `res` here. Would you mind fixing that as well?

Reply:
There is a [http_get_free(res); here](https://github.com/clibs/clib/pull/295/files#diff-adf3d2825fdcb98a17a9be70cb2b580b7c14ff08dc0039eaef032c0a07a9ce42R121)

Reply:
Oh. Forgot it is a return...I'll fix it.

Reply:
Done.